java/light-4j/src/main/resources/config/openapi-security.yml

```diff
@@ -0,0 +1,43 @@
+# Security configuration for openapi-security in light-rest-4j. It is a specific config
+# for OpenAPI framework security. It is introduced to support multiple frameworks in the
+# same server instance. If this file cannot be found, the generic security.yml will be
+# loaded for backward compatibility.
+---
+# Enable JWT verification flag.
+enableVerifyJwt: ${openapi-security.enableVerifyJwt:false}
+
+# Extract JWT scope token from the X-Scope-Token header and validate the JWT token
+enableExtractScopeToken: ${openapi-security.enableExtractScopeToken:true}
+
+# Enable JWT scope verification. Only valid when enableVerifyJwt is true.
+enableVerifyScope: ${openapi-security.enableVerifyScope:true}
+
+# Enable JWT scope verification. 
```

Comment:
trailing whitespace

Reply:
These files are configuration files to enable or disable cross-cutting concerns in the request/response chain. They are checked during the server startup and it won't impact the runtime performance as the security is not enabled for this performance test.